test/engine_fs_test.go

```diff
@@ -59,20 +59,44 @@ func (e *fsEngine) GetUID(user User) keybase1.UID {
 	return uid
 }
 
-func buildRootPath(u *fsUser, tlfName string, isPublic bool) string {
+func buildRootPath(u *fsUser, isPublic bool) string {
 	var path string
 	if isPublic {
-		path = filepath.Join(u.mntDir, "public", tlfName)
+		path = filepath.Join(u.mntDir, "public")
```

Comment:
Random thought, maybe not for this PR: Maybe we should consolidate all the `PublicName` consts into `libfs`, and we could use that here?  (Same with private.)